docs/accessibility.md

```diff
@@ -0,0 +1,89 @@
+* [W3C’s Web Accessibility Initiative](http://www.w3.org/WAI/): The World Wide Web Consortium's Web Accessibility Initiative provides strategies, guidelines and resources to make the Web accessible to people with disabilities.
+* [A11y Buzz](http://www.a11ybuzz.com/): Several web folks in the accessibility community launched this site in January 2012. It's a collection of accessibility resources that are triaged and reviewed by experts before they're posted.
+* [The Six Simplest Accessibility Tests Anyone Can Do](http://www.karlgroves.com/2013/09/05/the-6-simplest-web-accessibility-tests-anyone-can-do/): A handful of simple things you can do to think about and implement accessibility with little effort as you work on projects.
+* [Why Keyboard Accessibility Isn’t the Same Thing as Screen Reader Accessibility](https://www.ssbbartgroup.com/blog/why-keyboard-accessibility-isnt-the-same-thing-as-screen-reader-accessibility/): This blog post explains the three critical aspects of accessibility for interactive web technologies: Keyboard Accessibility, Screen Reader Accessibility and Cognitive Accessibility, and the differences between each.
+* [Teaching Accessibility Core Rules](http://www.deque.com/pragmatica11y-teaching-accessibility-core-rules): This post breaks down accessibility principles to their core, making them even easier to think about during projects.
+* Presentation: [Explorations in the Virtual DOM: How React.js Impacts Accessibility](https://marcysutton.github.io/react-a11y-presentation/)
+* [Inclusive Design Fundamentals](https://isner.github.io/inclusive-design-fundamentals/), especially the [disability types](https://isner.github.io/inclusive-design-fundamentals/handouts/disability-types.html) to learn about how different disabilities affect how people use technology.
+
+### Going in depth
+
+A set of blogs, written by accessibility consultants and evangelists that go in depth on all things accessibility.
+
+* [The Paciello Group Blog](http://www.paciellogroup.com/blog/): A blog about web accessibility by one of the first accessibility consulting firms.
+* [Simply Accessible](http://simplyaccessible.com/): A blog with in-depth web accessibility tutorials.
+* [WebAxe](http://www.webaxe.org/): A blog and podcast about web accessibility.
+* [456 Berea Street](http://www.456bereastreet.com/): A blog on accessibility and web development by Roger Johansson.
+* [Accessible Culture](http://www.accessibleculture.org/): A blog about web accessibility by Jason Kiss, an accessibility and standards researcher located in Wellington, New Zealand.
+
+## Tools
+
+Find tools that will help you bring accessibility into your workflow.
+
+### UX and Design
```

Comment:
a "thinking out loud" note - it would be great to see a designer/design team make a page like the [typography page](https://wpcalypso.wordpress.com/devdocs/typography) or [the design handbook colors page](https://wordpress.com/design-handbook/colors/), but up to date & with indication of which colors can be used as text color against which background.

Reply:
cc @megsfulton for the idea